### ISSUE TITLE
Gain understanding on Logisic Regression, k-NN, SVM and Perceptron algorithms, and loss functions (gradient descent)

### ISSUE THREAD

**Issue by Deniall** [CPSSD/LUCAS#62]
_In this order, a linear progression of learning can be gained to get a good foundational understanding of the most important ML technologies and concepts so we can progress into Deep Learning_
